app-backend/database/src/main/scala/com/azavea/rf/database/tables/Projects.scala

```diff
@@ -391,6 +391,8 @@ object Projects extends TableQuery(tag => new Projects(tag)) with LazyLogging {
                     ColorCorrect.Params(
                       redBand, greenBand, blueBand, // Bands
                       None, None, None,             // Gamma
+                      None, None, None,             // Clipping Max
```

Comment:
A little more description might be good here, though I guess it's maybe obvious that the order is RGB